repository/src/main/java/tech/ydb/yoj/repository/db/common/CommonConverters.java

```diff
@@ -79,18 +81,27 @@ public static <S> ThrowingGetter<S> stringValueGetter(Type type, Function<S, Obj
         };
     }
 
-    private static Method getStringValueDeserializerMethod(Class<?> clazz) {
+    private static ThrowingGetter<Object> getStringValueDeserializerMethod(Class<?> clazz) {
         for (String methodName : new String[]{"fromString", "valueOf"}) {
             try {
                 Method method = clazz.getMethod(methodName, String.class);
                 if (isStatic(method.getModifiers())) {
-                    return method;
+                    return s -> method.invoke(null, s);
                 }
             } catch (NoSuchMethodException ignored) {
             }
         }
+
+        try {
+            Constructor<?> constructor = clazz.getConstructor(String.class);
+            return s -> s != null ? constructor.newInstance(s) : null;
```

Comment:
`return constructor.newInstance(s)` is sufficient, `s` will never be `null`. Null values are **not** passed to converters